vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
well, no, it's because it won't work in .NET Core 👅 
This is related to WMI, which is not something we can support outside of Windows... not sure what I'd do with this test... It's probably better to keep it where it is and postpone this decision (as the most likely thing is that we should treat this as a .NET Framework only extension in its own project)... I believe those test don't even work today